.github/workflows/crates-builder.yaml

```
@@ -0,0 +1,59 @@
+name: Build workspace crates
+
+on:
+  workflow_call:
+    inputs:
+      crates:
+        required: true
+        default: "[]"
```

actionlint (1.7.12)

[error] input "crates" of workflow_call event has the default value "[]", but it is also required. if an input is marked as required, its default value will never be used

(events)